go.mod

```diff
@@ -10,19 +10,19 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/mattermost/ldap v3.0.4+incompatible // indirect
-	github.com/mattermost/mattermost-server/v6 v6.0.0-20211029033710-ec56cac42da7
+	github.com/mattermost/mattermost-server/v6 v6.0.0-20220204061906-8223968576c7
```

Comment:
Is this the hash to my PR branch (https://github.com/mattermost/mattermost-server/pull/19485/files)? Doesn't seem to match though :thinking: 

In any case, this is tricky to handle. We need to test along with my branch, and then merge the server PR, update the hash here, and merge the load-test PR.

Reply:
Yep, its pointing to your PR branch so I could get the new methods imported. I will update once your PR is merged.